### PR TITLE
Cloud sites image updates 160620 1

### PR DESCRIPTION
**Pull request by kylelaffoon**
Update verbiage and images in Getting Started With Cloud Sites: How To Add A New Website and Getting Started With Cloud Sites - Managing DNS Records articles.

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-731dc989a9/how-to/)

File: content/cloud-sites/getting-started-with-cloud-sites-how-to-add-a-new-website.md [hunk 1]
```diff
@@ -60,54 +49,42 @@ This article will show you how to create your domain through the [Cloud Sites Co
...
 
 If you get the message that *the domain cannot be automatically
 registered or transferred*, this means that the domain has **already
 been registered** and is not currently available to be transferred to
 the Rackspace Cloud. If you are the owner of the domain do not worry,
 **you can still proceed** with adding the domain to the Rackspace Cloud
-by clicking on **Continue**.
+by clicking **Continue**.
 
 ![]http://c806394.r94.cf2.rackcdn.com/noautotransfer.png)
```
Comment (stephamon) [inline content/cloud-sites/getting-started-with-cloud-sites-how-to-add-a-new-website.md]: This image is missing an open parenthesis after the brackets and before "http"

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cadf07ee93/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**